src/test/java/org/zalando/nakadi/controller/EventTypeControllerTest.java

```diff
@@ -626,7 +628,28 @@ public void whenOptionsRetentionTimeExist() throws Exception {
     }
 
     @Test
-    public void whenOptionsRetentionTimeBiggerThanMax() throws Exception {
+    public void whenGetOptionsRetentionTimeExist() throws Exception {
+        final EventType defaultEventType = buildDefaultEventType();
+        final EventTypeOptions eventTypeOptions = new EventTypeOptions();
+        eventTypeOptions.setRetentionTime(150L);
+        defaultEventType.setOptions(eventTypeOptions);
+
+        Mockito.doReturn(Collections.singletonList(defaultEventType)).when(eventTypeRepository).list();
+
+        getEventType()
```

Comment:
why you get here a list of of event types? I think it would be more straightforward if you get a single event type, then it will be possible to use something like
`.andExpect(jsonPath("$.options.retention_time", equalTo(150)))` instead of StringContains matching